src/Services/PlatformService.cs

```diff
@@ -42,7 +46,32 @@ private static Platform ParseOperatingSystem(UserAgent agent)
             return Platform.Others;
         }
 
-        private static Processor ParseProcessor(UserAgent agent, Platform os)
+        private static Version GetVersion(string agent, Platform platform) 
+            => platform switch
+        {
+            Platform.Unknown => new Version(),
+            Platform.Others  => new Version(),
+            Platform.Windows => ParseOsVersion(agent, "windowsnt"),
+            Platform.Android => ParseOsVersion(agent, "android"),
+            Platform.Mac     => ParseOsVersion(agent, "intelmacosx"),
+            Platform.iOS     => ParseOsVersion(agent, "cpuiphoneos"),
+            Platform.Linux   => ParseOsVersion(agent, "rv:"),
+            _                => new Version()
+        };
+
+        private static Version ParseOsVersion(string agent, string versionPrefix) =>
```

Comment:
I think this is not maintainable, while it is a working solution for the time being.
Maybe I will merge this PR first and rework on the code base later.